src/Components/Contributions/ItemsModal.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react';
+import { Modal, Button } from 'react-bootstrap'
+import ModalDialog from 'react-bootstrap/ModalDialog'
```

Comment:
you can write 

import {Modal Dialog, ModalHeader, ModalTitle} from 'react-bootstrap